frontend/src/components/module-editor/EditContentOptionsMenu.tsx

```diff
@@ -0,0 +1,44 @@
+import React from "react";
+import { HStack, IconButton } from "@chakra-ui/react";
+import { EditIcon, CopyIcon, DeleteIcon } from "@chakra-ui/icons";
+import { EditContentOptionsMenuProps } from "../../types/ModuleEditorTypes";
+
+const EditContentOptionsMenu = ({
+  isVisible,
+  onEditClick,
+  onCopyClick,
+  onDeleteClick,
+}: EditContentOptionsMenuProps): React.ReactElement => {
+  const currentTabIndex = isVisible ? 0 : -1;
+
+  return (
+    <HStack spacing={4} opacity={isVisible ? 1 : 0}>
+      <IconButton
+        aria-label="Edit this content block"
+        tabIndex={currentTabIndex}
```

Comment:
If we do the conditional rendering, we might also be able to avoid having to set the `tabIndex` manually.

Reply:
Discussed sync -- conditional rendering is not great for us since we want the text container to stay in the same place, no matter if we're rendering the options menu (aka when hovered). Instead, switch to using `visibility` attribute which will handle the tab index + opacity all in one go